transports/sqs/configuration-ooptions-policy-intro.include.md

```diff
@@ -0,0 +1,8 @@
+NServiceBus automatically subscribes to all event types an endpoint has handlers for. So given an endpoint with the following two handlers:
```

Comment:
The word 'So given' had me expecting a result directly after the listing. Given... then... 

Instead there is different text explaining autosubscribe.

Consider rewording?